src/components/PlioQuestion.vue

```diff
@@ -11,7 +11,10 @@
               {{ plioQuestion.item.question.text }}
             </div>
             <div class="close-container" id="skip-button" @click="clickSkip">
-              <i class="fas fa-window-close"></i>
+              <font-awesome-icon 
+                :icon="['fas', 'window-close']"
+                class="skip-icon"
+              ></font-awesome-icon>
```

Comment:
Before: an `i` tag was used with the specific class that defined that icon in the main css file that was imported
Now: Only a specific icon will be loaded into a component named `font-awesome-icon`